data/initrd/scripts/url/hmc/mount

```diff
@@ -0,0 +1,16 @@
+#! /bin/bash
+
+set -e
+
+echo "url_dir = '$url_dir'"
+
+modprobe hmcdrv
+
+if [ -d $url_dir ] ; then
+  hmcdrvfs $url_dir
```

Comment:
Is the program already included? s390utils-hmcdrvfs.rpm does not seem to be mentioned anywhere